docs/src/main/paradox/project/migration-guides.md

```diff
@@ -10,16 +10,29 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
 
 This is just stub documentation. It will be improved.
 
-* for Pekko jar dependencies, the groupId is "org.apache.pekko" instead of "com.typesafe.akka"
-* the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar 
+In your dependency management files:
+* Update dependency Akka version to the latest version of Pekko `1.0.x`.
+* for Pekko jar dependencies, the groupId is `org.apache.pekko` instead of `com.typesafe.akka` or `com.lightbend.akka.<package_name>`
+* the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar
 * Alpakka equivalent is "pekko-connectors" - e.g. pekko-connectors-kafka_2.13.jar instead of alpakka-kafka_2.13.jar
-* Pekko packages start with "org.apache.pekko" instead of "akka" - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
-* Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. PekkoException instead of AkkaException
-* Configs in `application.conf` use "pekko" prefix instead of "akka"
+
+In config files:
+* Configs use `pekko` prefix instead of `akka`
+
+In source code files:
+* Pekko packages start with `org.apache.pekko` instead of `akka` - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
+* Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. `PekkoException` instead of `AkkaException`
+* Configs use `pekko` prefix instead of `akka`
+
+In Documentation:
+* Where appropriate, text should refer to `Pekko` instead of `Akka`
+* URLs linking to files in https://doc.akka.io/docs/akka/ should instead point at https://pekko.apache.org/docs/pekko/
+
 * We have changed the default ports used by the pekko-remote module.
```

Comment:
this port information is useful but it is now sort of orphaned by the artificial breaking up of the migration info into categories

there are not so many bullet points that we need to split it up like this - especially when we then stick some of the bullet points off in 'no category'